samza-core/src/main/java/org/apache/samza/config/ClusterManagerConfig.java

```diff
@@ -189,12 +197,15 @@ public String getContainerManagerClass() {
     return get(CLUSTER_MANAGER_FACTORY, CLUSTER_MANAGER_FACTORY_DEFAULT);
   }
 
-  public boolean getJmxEnabled() {
+  public boolean getJmxEnabledOnJobCoordinator() {
     if (containsKey(CLUSTER_MANAGER_JMX_ENABLED)) {
+      log.info("Configuration {} is deprecated. Please use {}", CLUSTER_MANAGER_JMX_ENABLED, JOB_JMX_ENABLED);
```

Comment:
This message should be logged at the warn log-level.

Reply:
info is fine

Reply:
@shanthoosh is right, please use warn since this configuration may go away in future.